modules/builds-disabling-build-strategy-globally.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * assembly/builds
+// * builds/securing-builds-by-strategy.adoc
+
+
+[id='builds-disabling-build-strategy-globally-{context}']
+= Disabling access to a build strategy globally
+
+To prevent access to a particular build strategy globally, log in as a user with
+*cluster-admin* privileges, remove the corresponding role from the
+*system:authenticated* group, and apply the annotation
+`openshift.io/reconcile-protect: "true"` to protect them from changes between
+the API restarts. The following example shows disabling the docker build
+strategy.
+
+.Procedure
+
+. Apply the `openshift.io/reconcile-protect` annotation:
++
+----
+$ oc edit clusterrolebinding system:build-strategy-docker-binding
+
+apiVersion: v1
+groupNames:
+- system:authenticated
+kind: ClusterRoleBinding
+metadata:
+  annotations:
+    openshift.io/reconcile-protect: "true" <1>
```

Comment:
double check this procedure with @openshift/sig-auth

Reply:
@enj would you mind confirming the procedure, please? Thanks!